cmd/flags.go

```diff
@@ -24,6 +24,7 @@ var (
 	DefaultProcessType         = "web"
 	DefaultProjectMetadataPath = filepath.Join(".", "project-metadata.toml")
 	DefaultReportPath          = filepath.Join(".", "report.toml")
+	DefaultStackGroupPath      = ""
```

Comment:
I think this is a bit weird. If we have no default we don't need this var. We can treat it like `-cache-dir` and other inputs that have no default.